app/views/user/teams/_form.html.erb

```diff
@@ -0,0 +1,12 @@
+<%= form_tag(user_teams_path, method: :put) do %>
+  <div class="field-wrapper">
+    <%= label_tag :team_slug, 'Teams', class: 'col-sm-2 control-label' %>
+    <div class="col-sm-10">
+      <%= select_tag :team_slug, options_from_collection_for_select(@teams, 'slug', 'name'), disabled: @teams.blank? %>
```

Comment:
I'm not sure but I think you can use `:slug` and `:name` here instead of their string versions.